tests/runtest.proj

```diff
@@ -255,7 +255,7 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
                     {
                         sErrorText = System.IO.File.ReadAllText(errorFile)%3B
                     }
-                    catch(Exception ex)
```

Comment:
Please include this in the message. This is a test and as such we _need_ the exception details. It is easy to address:

>sErrorText = $"Unable to read error file: {errorFIle}\n{ex}"%3B

Please also update the logic in the other location for the exception.